node/src/chain/acceptor.rs

```diff
@@ -294,18 +294,16 @@ impl<DB: database::DB, VM: vm::VMExecution, N: Network> Acceptor<N, DB, VM> {
         let generator = generator
             .try_into()
             .map_err(|e| anyhow::anyhow!("Cannot deserialize bytes {e:?}"))?;
+        // FIX ME: We should track voters reward too
```

Comment:
Is reward tracking in this context actually relevant? If we keep the provisioner list just for sortition, rewards have no influence.
If instead we use this info to actually update the state, than this FIXME is kind of important...isn't it?